arches/app/etl_modules/save.py

```diff
@@ -11,115 +11,131 @@
 logger = logging.getLogger(__name__)
 
 
+def save_to_tiles(userid, loadid):
+    with connection.cursor() as cursor:
+        disable_tile_triggers(cursor, loadid)
+        error_saving_tiles = _save_to_tiles(cursor, loadid)
+        reenable_tile_triggers(cursor, loadid)
+        if error_saving_tiles:
+            return error_saving_tiles
+
+        _post_save_edit_log(cursor, userid, loadid)
```

Comment:
This is the only change to this file: decomposing save_to_tiles() into functions that can be reordered as the JSON-LD importer needs to. Changing the indent level caused an ugly diff, but that's all that's going on below.